### ISSUE TITLE
Spark v2 - DDO & Allocators

### ISSUE THREAD

**Issue by bajtos** [space-meridian/roadmap#115]
**TL;DR: Each miner storing publicly retrievable data should show on the Spark public dashboard.**

Spark is checking whether Storage Providers are offering public data retrievals for deals that are expected to be retrievable. To do so, we need to understand which deals are expected to be retriable. Earlier this year, the introduction of Direct Data Onboarding (DDO) and FIL+ Allocator Pathways significantly changed the Filecoin landscape, and our previous solution is no longer sufficient. The goal of this milestone is to improve Spark to support DDO & Allocators.

Resources:
- https://www.notion.so/spacemeridian/Spark-v2-40138e6879a04e8589925b3ee9ae11b1
- https://docs.google.com/document/d/1YvFYYUwP6VibKyTeHLd89mrkB1S5sVjq9ydFCGbBfrw/edit

**Comment by MikeH1999** [comment]
Sorry, I'm wondering how long this will take.

**Comment by juliangruber** [comment]
It's currently planned to land this year https://github.com/space-meridian/roadmap/issues/113
